modules/developer_manual/pages/app/fundamentals/filesystem.adoc

```diff
@@ -110,3 +110,12 @@ class AuthorStorage {
     }
 }
 ----
+
+== How to Get the Storage Owner Using a File Id
+
+A storage's owner can be retrieved using a file id, as in the following example.
+
+[source,php]
+----
+include::example$app/fundamentals/filesystem/getOwnerByFileId.php[]
```

Comment:
I think the link contains an error:
`include::example$app/fundamentals/filesystem/getOwnerByFileId.php[]`
imho: `example$app` --> `example/app`